docs/source/reference/integration.rst

```diff
@@ -148,3 +148,62 @@ XGBoost
+| `ChainerMN <https://github.com/optuna/optuna/blob/master/optuna/integration/chainermn.py>`_                                                                                       | chainermn                          |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------+
+| `Chainer <https://github.com/optuna/optuna/blob/master/optuna/integration/chainer.py>`_                                                                                           | chainer                            |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------+
+| `pycma <https://github.com/optuna/optuna/blob/master/optuna/integration/cma.py>`_                                                                                                 | cma                                |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------+
+| `Dask <https://github.com/optuna/optuna/blob/master/optuna/integration/dask.py>`_                                                                                                 | distributed                        |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------+
+| FastAI (`v1 <https://github.com/optuna/optuna/blob/master/optuna/integration/fastaiv1.py>`_, `v2 <https://github.com/optuna/optuna/blob/master/optuna/integration/fastaiv2.py>`_) | fastai                             |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------+
+| `Keras <https://github.com/optuna/optuna/blob/master/optuna/integration/keras.py>`_                                                                                               | keras                              |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------+
+| `LightGBM <https://github.com/optuna/optuna/blob/master/optuna/integration/lightgbm.py>`_                                                                                         | lightgbm, sklearn                  |
```

Comment:
I think that `sklearn` is not necessary.

Reply:
This part imports `sklearn`, so we need `sklearn` if my understanding is correct.
Btw, I noticed that only this line says `sklearn` rather than `scikit-learn`, so I will replace it with `scikit-learn`.
https://github.com/optuna/optuna/blob/master/optuna/integration/_lightgbm_tuner/optimize.py

Reply:
Ah, OK. How about splitting the row with `LightGBMPruningCallback` and `LightGBMTuner`?

Reply:
I did it in the latest commit:)